src/commands/actions.js

```diff
@@ -43,7 +43,7 @@ module.exports = function(vorpal, broker) {
 				if (args.options.filter && !match(item.name, args.options.filter))
 					return;
 
-				const serviceName = item.name.split(".")[0];
+				const serviceName = item.name.split(".").slice(0, -1).join(".");
```

Comment:
I've changed the code because it gives better result if the service name contains other "." dots, e.g.:
![image](https://user-images.githubusercontent.com/306521/115139273-3d774380-a031-11eb-8859-a4ed34f828bc.png)